tests/test_viewport.py

```diff
@@ -315,6 +315,42 @@ def execute(self):
         assert self.py("print(vim.current.buffer)", regex="<buffer taskwiki.")
 
 
+class TestViewportInspectionWithVisibleTag(MultiSyntaxIntegrationTest):
+
+    viminput = """
+    HEADER2(Work tasks | +work +VISIBLE)
+
+    HEADER2(Home tasks | +home)
+    * [ ] tag work task  #{uuid}
+    """
+
+    vimoutput = """
+    ViewPort inspection:
+    --------------------
+    Name: Work tasks
+    Filter used: -DELETED -PARENT ( +work )
+    Defaults used: tags:['work']
+    Ordering used: status+,end+,due+,priority-,project+
+    Matching taskwarrior tasks: 0
+    Displayed tasks: 1
+    Tasks to be added:
+    Tasks to be deleted: tag work task
```

Comment:
Is this intended?

I'd appreciate a comment explaining what this test really tests, as I don't think it's obvious. :-/